### ISSUE TITLE
Design guidelines

### ISSUE THREAD

**Issue by fedestyla** [pieroit/cheshire-cat#148]
To easy the development and avoid going in too many different directions, I think it could be good for the project to integrate some kind of design guidelines.

I'd think a potential solution would be to integrate material design 3 and use ready-made design kit from Figma -> https://material.io/blog/material-3-figma-design-kit

**Comment by fedestyla** [comment]
The idea is that they can be exported from Figma as plain CSS or as React components via plugin. The benefit would be that we just use these bunch of widgets and we forget about coloring/styling and so on as we would follow plain material design v3! Probably not super important at the moment but would be good to have super clear design guidelines!

**Comment by antonioru** [comment]
@fedestyla good point, this is something I've been trying to introduce for a bit. 
I think we need to agree on a design system and I was looking into Chakra-UI to be honest, but I'm open to anything that will speed up the development of the FE
